docs/releases/v3.0.1.rst

```diff
@@ -0,0 +1,68 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2018 CERN.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+Version 3.0.1
+=============
+
+Invenio v3.0.1 fixes several bugs in Invenio v3.0.0.
+
+Bug fixes
+---------
+
+- **Invenio-Access:** Fixed bug with the ``any_user``-need not being provided
+  by the anonymous identity.
+
+- **Invenio-App:** Fixed compatibility issue with latest release of
+  Flask-Talisman.
+
+- **Invenio-DB:** Fixed a compatibility issue with newly released
+  SQLAlchemy-Continiuum.
+
+- **Invenio-Search:** Fixed a compatibility issue with
+  elasticsearch-dsl>=6.2.0.
+
+- **Pytest-Invenio:** Fixed a problem which caused Invenio modules to be loaded
+  despite the related test fixtures not being used, thus causing import errors
+  if the module was not installed.
+
+- **Pytest-Invenio:** Fixed a compatibility issue with pytest v3.8.x.
```

Comment:
To be more precise, it was starting with pytest 3.8.1 (https://github.com/pytest-dev/pytest/issues/3941) (3.8.0 was fine) :nerd_face: 

```suggestion
- **Pytest-Invenio:** Fixed a compatibility issue with pytest v3.8.1+.
```